.azure-pipelines.yml

```diff
@@ -479,7 +479,7 @@ jobs:
        eq(variables['Build.SourceBranch'], 'refs/heads/triggers/safari_stable'),
        and(eq(variables['Build.Reason'], 'Manual'), variables['run_all_safari']))
   strategy:
-    parallel: 5 # chosen to make runtime ~2h
+    parallel: 8 # chosen to make runtime ~2h
```

Comment:
Is the comment still true after this bump?

Reply:
Taking a recent complete run, maybe we should go for 7, but I was trying to leave a bit more of a margin this time?

Reply:
OK, sounds like you've looked at the total runtime at least. Thanks!